### PR TITLE
Add Move Transition OBS plugin

### PR DESCRIPTION
**Pull request by exeldro**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: com.obsproject.Studio.Plugin.MoveTransition.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+id: com.obsproject.Studio.Plugin.MoveTransition
+branch: stable
+runtime: com.obsproject.Studio
+runtime-version: stable
+sdk: org.kde.Sdk//5.15
```
Comment (hfiguiere) [inline com.obsproject.Studio.Plugin.MoveTransition.yaml:5]:
```suggestion
sdk: org.kde.Sdk//5.15-21.08
```

OBS use 5.15-21.08

File: flathub.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+    "skip-icons-check": true,
+    "only-arches": ["x86_64", "i386"],
```
Comment (hfiguiere) [inline flathub.json:3]:
```suggestion
    "only-arches": ["x86_64"],
```

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by hfiguiere** [comment]
@GeorgesStavracas this up your alley.

**Comment by GeorgesStavracas** [comment]
Sorry, this fell through the cracks. I'll have a look tonight!

File: com.obsproject.Studio.Plugin.MoveTransition.metainfo.xml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+
```
Comment (GeorgesStavracas) [inline com.obsproject.Studio.Plugin.MoveTransition.metainfo.xml:1]: Tiny nitpick: please remove this empty newline

File: com.obsproject.Studio.Plugin.MoveTransition.yaml [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+modules:
+  - name: move-transition
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DBUILD_OUT_OF_TREE=On
+    post-install:
+      - install -Dm644 --target-directory=${FLATPAK_DEST}/share/metainfo ${FLATPAK_ID}.metainfo.xml
+      - appstream-compose --basename=${FLATPAK_ID} --prefix=${FLATPAK_DEST} --origin=flatpak
+        ${FLATPAK_ID}
```
Comment (GeorgesStavracas) [inline com.obsproject.Studio.Plugin.MoveTransition.yaml:20]: I think this should be a single line command

**Review by GeorgesStavracas — CHANGES_REQUESTED** [review]
Two small comments, LGTM other than that. I think it can land after these comments are addressed.

**Comment by exeldro** [comment]
Applied suggested changes

**Review by GeorgesStavracas — APPROVED** [review]
LGTM. Please squash all commits with the first one, then let's land this \o/

**Pull request closed without merging**